src/pages/index.js

```diff
@@ -8,7 +8,7 @@ const IndexPage = () => (
   <div>
     <div className="index__splash">
       <h1 className="index__headline">
-        Web Audio Conf | Berlin
+        Web Audio Conf<span className="index__headline__delimiter"> | </span>Berlin
```

Comment:
Now you just need to change this one to `index__headlineDelimiter` :)

Reply:
Thanks for catching that stupid mistake.